xds/src/main/java/io/grpc/xds/XdsServerBuilder.java

```diff
@@ -41,10 +45,22 @@
 @ExperimentalApi("https://github.com/grpc/grpc-java/issues/7514")
 public final class XdsServerBuilder extends ForwardingServerBuilder<XdsServerBuilder> {
 
+  public static final Attributes.Key<AtomicReference<FilterChainSelector>>
+          ATTR_FILTER_CHAIN_SELECTOR_REF = Attributes.Key.create(
+          "io.grpc.xds.XdsServerBuilder.filterChainSelectorRef");
+  private static final long RETRY_DELAY_NANOS = TimeUnit.MINUTES.toNanos(1);
+
   private final NettyServerBuilder delegate;
   private final int port;
   private XdsServingStatusListener xdsServingStatusListener;
   private AtomicBoolean isServerBuilt = new AtomicBoolean(false);
+  private final AtomicReference<FilterChainSelector> filterChainSelectorRef =
```

Comment:
It seems this doesn't need to be a field. Move it to build()?

Reply:
k.